fastestimator/op/tensorop/loss/cross_entropy.py

```diff
@@ -17,7 +17,9 @@
 import tensorflow as tf
 import torch
 
-from fastestimator.backend import binary_crossentropy, categorical_crossentropy, sparse_categorical_crossentropy
+from fastestimator.backend.binary_crossentropy import binary_crossentropy
```

Comment:
all of these backend functions are at same level as the backend __init__ file, so they should import directly form fe.backend

Reply:
done